Jellyfin.Server/Jellyfin.Server.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>jellyfin</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- We need C# 7.1 for async main-->
+    <LangVersion>latest</LangVersion>
```

Comment:
Can we pin this 7.1 here, just to be safe?

Reply:
Safe of what?

Reply:
Safe of some c# 21 coming in sometime in the future and breaking badly our stuff...

Reply:
The default is the latest major version, `latest` just means latest minor version, I doubt the language spec will ever breaks backwards compatibility and certainly not in a minor release.

Reply:
You know better than I, so you win here :)